src/App.tsx

```diff
@@ -1,8 +1,49 @@
 import React from 'react';
 import './App.css';
+import { getPrepearedTodos } from './api';
+import { TodoList } from './TodoList';
+import { Todo } from './types';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+const App: React.FC = () => {
+  const [todos, setTodos] = React.useState<Todo[]>([]);
+  const [isLoaded, setLoading] = React.useState<boolean>(false);
+  const [isError, setError] = React.useState<string>('');
```

Comment:
- don't specify primitive types
- error is a string, why `isError`
```suggestion
  const [isLoaded, setLoading] = React.useState(false);
  const [error, setError] = React.useState('');
```